lib/marin/src/marin/download/huggingface/download_hf.py

```diff
@@ -20,25 +20,21 @@
 
 import logging
 import os
-import time
 import random
+import time
 
 import fsspec
-import ray
 from huggingface_hub import HfFileSystem
-from tqdm_loggable.tqdm_logging import tqdm_logging
-
-from marin.core.runtime import simple_backpressure
 from marin.download.huggingface.download import DownloadConfig
 from marin.utilities.validation_utils import write_provenance_json
+from zephyr import Dataset, flow_backend
 
-# Set up logging
-logger = logging.getLogger("ray")
+logger = logging.getLogger(__name__)
```

Comment:
fwiw there is a reason for "ray": ray privileges those logs and it seems to go out of its way to suppress the others (though you can get them)